classes/form/FormBuilderVocabulary.inc.php

```diff
@@ -476,6 +476,9 @@ function _smartyFBVTextArea($params, &$smarty) {
 				case 'variables':
 					$smarty->assign('FBV_' . $key, $value);
 					break;
+				case 'variablesType':
+					$smarty->assign('FBV_' . $key, $value);
```

Comment:
No need to implement this again -- you can add `variablesType` to the list above and it'll fall to the implementation in line 477.